src/main/kotlin/com/ampnet/mailservice/service/impl/UserMailServiceImpl.kt

```diff
@@ -110,32 +144,40 @@ class UserMailServiceImpl(
         invitationMail.setTemplateData(request.organizationName, request.coop).setLanguage(senderResponse.language)
             .sendTo(request.emails.toList()) { failedMails ->
                 val filedMailRecipients = failedMails.map { it.allRecipients.toString() }
-                failedDeliveryMail.setTemplateData(filedMailRecipients).setLanguage(senderResponse.language)
-                    .sendTo(senderResponse.email)
+                failedDeliveryMail.setTemplateData(filedMailRecipients).setCoop(request.coop)
+                    .setLanguage(senderResponse.language).sendTo(senderResponse.email)
             }
     }
 
     override fun sendDepositRequestMail(user: UUID, amount: Long) {
         val userResponse = getUser(user)
-        depositRequestMail.setTemplateData(amount).setLanguage(userResponse.language).sendTo(userResponse.email)
+        depositRequestMail.setTemplateData(amount).setCoop(userResponse.coop)
+            .setLanguage(userResponse.language).sendTo(userResponse.email)
     }
 
     override fun sendDepositInfoMail(user: UUID, minted: Boolean) {
         val userResponse = getUser(user)
         val hasProjectWhichCanReceiveInvestment = if (minted.not()) false
         else hasProjectWhichCanReceiveInvestment(userResponse.coop)
-        depositMail.setTemplateData(userResponse.coop, minted, hasProjectWhichCanReceiveInvestment)
+        when {
+            (minted && hasProjectWhichCanReceiveInvestment) -> depositMail
+            (minted && hasProjectWhichCanReceiveInvestment.not()) -> depositNoProjectInvestmentMail
+            else -> depositInfoFailedMail
+        }.setTemplateData(userResponse.coop)
             .setLanguage(userResponse.language).sendTo(userResponse.email)
     }
 
     override fun sendWithdrawRequestMail(user: UUID, amount: Long) {
         val userResponse = getUser(user)
-        withdrawRequestMail.setTemplateData(amount).setLanguage(userResponse.language).sendTo(userResponse.email)
+        withdrawRequestMail.setTemplateData(amount).setCoop(userResponse.coop)
+            .setLanguage(userResponse.language).sendTo(userResponse.email)
     }
 
     override fun sendWithdrawInfoMail(user: UUID, burned: Boolean) {
         val userResponse = getUser(user)
-        withdrawInfoMail.setTemplateData(burned).setLanguage(userResponse.language).sendTo(userResponse.email)
+        val mail = if (burned) withdrawInfoMail.setCoop(userResponse.coop)
+        else withdrawFailedInfoMail
```

Comment:
```suggestion
        val mail = if (burned) withdrawInfoMail.setCoop(userResponse.coop) else withdrawFailedInfoMail
```
What is the value for intellij preferences -> editor -> code style -> kotlin -> wrapping -> hard wrap at? It should be 120.

Reply:
It's 120.